### DISCUSSION TITLE
illegal_argument_exception

### DISCUSSION THREAD

**Discussion by Cbudano** [toluaina/pgsync#477] [Q&A]
hi, im trying to configure pgsync to insert into aws opensearch.
i was able to make the conection but im getting:

elasticsearch.exceptions.RequestError: RequestError(400, 'illegal_argument_exception', 'Action/metadata line [1] contains an unknown parameter [_type]')

it seems _type got depredicated, is any configuration i can set to avoid this error?

**Comment by bbirec** [comment]
You should set environment variables **BOTH** `ELASTICSEARCH=false` and `OPENSEARCH=true`.
